worker/echo/main.go

```diff
@@ -16,6 +16,16 @@ import (
 var yggdDispatchSocketAddr string
 
 func main() {
+	// Set log level inherited from the environment
+	if logLevel, ok := os.LookupEnv("YGG_LOG_LEVEL"); ok {
+		level, err := log.ParseLevel(logLevel)
+		if err != nil {
+			log.Fatalf("cannot parse log level: %v", err)
```

Comment:
When log level could not be parsed, is it good idea to set this wrong log level. Shouldn't we skip skip setting log level in this case?

Reply:
`log.Fatal` will print the message string and then exit the program, so here we are not setting an invalid level. The program is exiting.

Reply:
Oh, I didn't know that. It is then OK.